scripts/freesound_download_resample/download_resample_freesound.sh

```diff
@@ -0,0 +1,117 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/bin/bash
+
+# This is bash script actually run the downloading and resampling script.
+# See instructions in freesound_download.py
+
+# Change this arguments if you want
+page_size=100   # Number of sounds per page
+max_samples=200 # Maximum number of sound samples
+min_filesize=0  # Maximum filesize allowed (in MB)
```

Comment:
I assume the comment here should read "Minimum" :)

Reply:
haha 😆